### PR TITLE
added how to create data rows with embeddings

### PR DESCRIPTION
**Pull request by ovalle15**

### CODE REVIEW

File: examples/basics/custom_embeddings_sdk.ipynb [hunk 1]
```diff
@@ -39,7 +39,7 @@
     {
```
Comment (Gabefire) [inline examples/basics/custom_embeddings_sdk.ipynb:39]: <p>Can you seperate out the dataset id in a different cell like in this notebook this: <a href="https://github.com/Labelbox/labelbox-python/blob/develop/examples/basics/datasets.ipynb" target="_blank">https://github.com/Labelbox/labelbox-python/blob/develop/examples/basics/datasets.ipynb</a></p><p> </p><p>Also side note while we are at it in that notebook I linked can you remove the dataset id</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1582/discussion/#comment-1585294607'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (Gabefire) [inline examples/basics/custom_embeddings_sdk.ipynb:39]: <p>Can you remove the line above emeddings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1582/discussion/#comment-1585294609'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (ovalle15) [reply examples/basics/custom_embeddings_sdk.ipynb:39]: Done

Comment (ovalle15) [reply examples/basics/custom_embeddings_sdk.ipynb:39]: Done

**Review by Gabefire — APPROVED** [review]

**Pull request merged**